components/dashboard/src/data/prebuilds/prebuild-logs-emitter.ts

```diff
@@ -145,29 +145,40 @@ function streamPrebuildLogs(
         };
 
         let response: Response | undefined = undefined;
+        let abortController: AbortController | undefined = undefined;
         let reader: ReadableStreamDefaultReader<Uint8Array> | undefined = undefined;
         try {
+            abortController = new AbortController();
+            disposables.push({
+                dispose: async () => {
+                    abortController?.abort();
+                    await reader?.cancel();
+                },
+            });
             console.debug("fetching from streamUrl: " + streamUrl);
             response = await fetch(streamUrl, {
                 method: "GET",
-                cache: "no-cache",
```

Comment:
using `no-cache` here might've been problematic, because the browser makes a conditional request in the case there is a cache entry present for this request. The problem is that no-cache actually does cache 🙃